src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Specialized;
+using FluentValidation.Results;
+using NLog;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Configuration;
+
+namespace NzbDrone.Core.Notifications.Notifiarr
+{
+    public interface INotifiarrProxy
+    {
+        void SendNotification(StringDictionary message, NotifiarrSettings settings);
+        ValidationFailure Test(NotifiarrSettings settings);
+    }
+
+    public class NotifiarrProxy : INotifiarrProxy
+    {
+        private const string URL = "https://notfiarr.com";
+        private readonly IHttpClient _httpClient;
+        private readonly IConfigFileProvider _configFileProvider;
+        private readonly Logger _logger;
+
+        public NotifiarrProxy(IHttpClient httpClient, IConfigFileProvider configFileProvider, Logger logger)
+        {
+            _httpClient = httpClient;
+            _configFileProvider = configFileProvider;
+            _logger = logger;
+        }
+
+        public void SendNotification(StringDictionary message, NotifiarrSettings settings)
+        {
+            try
+            {
+                ProcessNotification(message, settings);
+            }
+            catch (NotifiarrException ex)
+            {
+                _logger.Error(ex, ex.Message);
+                throw new NotifiarrException(ex.Message);
```

Comment:
If we want to throw the exception, just `throw` would accomplish that without including this in the stack trace, though this might also negatively affect the exception the caller receives as well.